### ISSUE TITLE
Publishing components

### ISSUE THREAD

**Issue by debopamsengupta** [opentable/oc#318]
OC version: 0.33.22
node: 6.5.0

When trying to publish a new component to a registry, I get  the following error:
```
An error happened when publishing the component: TypeError: targz.extract is not a function
    at module.exports (/usr/src/app/node_modules/oc/src/registry/domain/extract-package.js:16:9)
    at /usr/src/app/node_modules/oc/src/registry/routes/publish.js:43:5
    at callbacks (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:164:37)
    at options.beforePublish (/usr/src/app/node_modules/oc/src/registry/domain/options-sanitiser.js:14:55)
    at callbacks (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:164:37)
    at param (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:138:11)
    at param (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:135:11)
    at param (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:135:11)
    at pass (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:145:5)
    at Router._dispatch (/usr/src/app/node_modules/oc/node_modules/express/lib/router/index.js:173:5)
```

The documentation for the package ```targz``` (https://www.npmjs.com/package/targz) says to use the function ```decompress``` instead of ```extract```

Am I missing something ? Any help appreciated :)

**Comment by matteofigus** [comment]
Good catch I think this bug was introduced with 0.33.31 https://github.com/opentable/oc/pull/316

I'll have a look now. Thanks

**Comment by debopamsengupta** [comment]
Hi @matteofigus thanks for the quick response, will this be released as a new version soon ? :)

**Comment by matteofigus** [comment]
@debopamsengupta just released this with `0.33.23` ;) Thanks for your help!
